### PR TITLE
Place details page

### PR DESCRIPTION
**Pull request by ajones55555**

### CODE REVIEW

File: src/pages/place/[place_id]/index.tsx
Comment (elenirotsides) [inline src/pages/place/[place_id]/index.tsx:40]: Do we wanna wait until the package is ready before we merge this in?

**Comment by elenirotsides** [comment]
If there isn't a rating at all, I don't think `resources` should be red since no rating !== bad rating 

image

@aej11a

File: src/pages/place/[place_id]/index.tsx
Comment (ajones55555) [reply src/pages/place/[place_id]/index.tsx:40]: nah

Comment (ajones55555) [reply src/pages/place/[place_id]/index.tsx:40]: typescript is mostly useful during development, so if we're adding things that should ADD new types then we would block them from merge, but i wouldn't block something that is just going to USE types if the dev work is already done

Comment (elenirotsides) [reply src/pages/place/[place_id]/index.tsx:40]: gotcha!

**Review by dcarpenter31 — APPROVED** [review]

**Review by elenirotsides — APPROVED** [review]

**Pull request merged**
